charts/opencloud/templates/onlyoffice/deployment.yaml

```
@@ -25,9 +25,9 @@ spec:
           command: ["/bin/sh", "/entrypoint-override.sh"]
           env:
             - name: WOPI_ENABLED
-              value: "{{ .Values.onlyoffice.wopi.enabled }}"
+              value: {{ tpl (toString .Values.onlyoffice.wopi.enabled) . | quote }}
             - name: USE_UNAUTHORIZED_STORAGE
-              value: "{{ .Values.onlyoffice.useUnauthorizedStorage }}"
+              value: {{ tpl (toString .Values.onlyoffice.useUnauthorizedStorage) . | quote}}
```

🔍 Missing space before closing braces (helm lint and helm template work fine without it, just a style issue)

```suggestion
              value: {{ tpl (toString .Values.onlyoffice.useUnauthorizedStorage) . | quote }}
```

---

Thanks, fixed.